container/root/run.sh

```diff
@@ -37,6 +37,23 @@ else
     echo 'opcache.validate_timestamps=0' >> /etc/php5/mods-available/opcache.ini
   fi
 
+  if [[ $CFG_APP_MAX_BODY_SIZE ]]
```

Comment:
hm. maybe don' t call it `CFG_APP` (its for the webserver specifically)? `CFG_` gets transposed to env variables for php